README.md

```diff
@@ -79,6 +79,18 @@ class Search < ActiveRecord::Base
 end
 ```
 
+## When I query that model with `find` I get an error. What gives?
+
+Your view cannot have a primary key, but ActiveRecord's `find` method expects to
+query based on one. You can use `find_by!` or you can explicitly set the primary
+key column on your model like so:
+
+```ruby
+class People < ActiveRecord::Base
+  primary_key :id
```

Comment:
I think it's `self.primary_key = :id`.

http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/PrimaryKey/ClassMethods.html